apps/frontend-manage/src/components/evaluation/BarChart.tsx

```diff
@@ -59,7 +64,16 @@ function BarChart({ data, showSolution }: BarChartProps): React.ReactElement {
             fill: 'black',
             offset: 30,
             stroke: 'black',
-            style: { fontSize: '2rem' },
+            style: {
+              fontSize:
```

Comment:
Eher per Objekt/Map lösen (skalierbar für mehr Grössen)

Reply:
This was solved with a different resolver logic (same for the ones below)